.circleci/config.yml

```
@@ -234,7 +234,7 @@ jobs:
             pip install --no-cache-dir --upgrade -e .[dev,testing] --progress-bar off
             git clone --depth 1 https://github.com/plotly/dashR.git -b dev dashR
             git clone --depth 1 https://github.com/plotly/dash-html-components.git
-            git clone --depth 1 https://github.com/plotly/dash-core-components.git
+            git clone -b persistence-hg --depth 1 https://github.com/plotly/dash-core-components.git
```

Should be updated to point to the new branch?
```suggestion
            git clone -b hg-700-persistence--depth 1 https://github.com/plotly/dash-core-components.git
```